helm-charts/konk-service/templates/delete-apiservice-deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "konk-service.fullname" . }}-delete-apiservice
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "konk-service.labels" . | nindent 4 }}
+  annotations:
+    helm.sh/hook: pre-delete
+spec:
+  serviceAccountName: {{ include "konk-service.serviceAccountName" . }}
+  containers:
+  - name: kind
+    securityContext:
+      {{- toYaml .Values.kind.securityContext | nindent 10 }}
+    image: "{{ .Values.kind.image.repository }}:{{ .Values.kind.image.tag | default .Chart.AppVersion }}"
+    imagePullPolicy: {{ .Values.kind.image.pullPolicy }}
+    command: ["/bin/bash", "-c"]
+    args:
+      - |
+        /mounts/delete-api-service.sh
+    env:
+      - name: SERVICENAME
+        value: {{ .Values.service.name }}
+      - name: NAMESPACE
+        value: {{ .Release.Namespace }}
+      - name: KUBECONFIG
+        value: /etc/kubernetes/admin.conf
+    resources:
+      {{- toYaml .Values.kind.resources | nindent 10 }}
+    volumeMounts:
+    - name: kubeconfig
+      mountPath: "/etc/kubernetes"
+      readOnly: true
+    - name: ca-cert
+      mountPath: "certs"
+      readOnly: true
+    - name: mounts
+      mountPath: "mounts"
+      readOnly: true
+    - name: gen
+      mountPath: "gen"
+    readinessProbe:
+      exec:
+        command:
+        - cat
+        - /tmp/healthy
```

Comment:
Looks like it never finds this file, so never becomes ready. I don't see any code that would create it.